src/vma/dev/ib_ctx_handler.cpp

```diff
@@ -54,84 +54,36 @@
 #define ibch_logfuncall         __log_info_funcall
 
 
-ib_ctx_handler::ib_ctx_handler(struct ib_ctx_handler_desc *desc) :
-	m_flow_tag_enabled(false)
+ib_ctx_handler::ib_ctx_handler(ib_ctx_handler_desc *desc) :
+	  m_p_ibv_device(NULL)
+	, m_p_ibv_context(NULL)
+	, m_p_ibv_device_attr(NULL)
+	, m_p_ibv_pd(NULL)
+	, m_flow_tag_enabled(false)
 	, m_on_device_memory(0)
 	, m_removed(false)
 	, m_lock_umr("spin_lock_umr")
 	, m_umr_cq(NULL)
 	, m_umr_qp(NULL)
 	, m_p_ctx_time_converter(NULL)
+	, m_is_ok(false)
```

Comment:
Having this flag does not guarantee correct check of class object creation. Exception can happen during initialization of one of the objects in initialization list. In this case `m_is_ok` can be in undefined state.

Reply:
all members get NULL and 0, if you want i can initialize the lick in the constractor body.